packages/react/src/patterns/blocks/ContentBlockSimple/ContentBlockSimple.js

```diff
@@ -0,0 +1,94 @@
+/**
+ * Copyright IBM Corp. 2016, 2018
+ *
+ * This source code is licensed under the Apache-2.0 license found in the
+ * LICENSE file in the root directory of this source tree.
+ */
+
+import { ContentBlock } from '../../sub-patterns/ContentBlock';
+import { ContentItem } from '../../sub-patterns/ContentItem';
+import { settings as ddsSettings } from '@carbon/ibmdotcom-utilities';
+import { Image } from '../../../components/Image';
+import PropTypes from 'prop-types';
+import React from 'react';
+import { settings } from 'carbon-components';
+
+const { stablePrefix } = ddsSettings;
+const { prefix } = settings;
+
+/**
+ * Content Block - Simple pattern
+ *
+ * @param {object} props props object
+ * @param {string} props.copy Content block simple short copy to support the heading
+ * @param {string} props.linkType link type ( simple | jump | card )
+ * @param {object} props.link link object which includes url, link text and target properties.
+ * @param {string} props.heading Content block simple heading
+ * @param {object} props.cta cta object
+ * @param {object} props.contentGroup chilren content element
+ * @param {string} props.mediaType Media type, video or image
+ * @param {object} props.mediaData Data for renderimg the media
+ * @param {Array} props.items Array of objects with data for ContentItems
+ * @returns {*} Content block simple pattern
+ */
+const ContentBlockSimple = ({
+  copy,
+  heading,
+  items,
+  mediaType,
+  mediaData,
+  cta,
+}) => (
+  <ContentBlock
+    heading={heading}
+    copy={copy}
+    items={items}
+    cta={cta}
+    style
+    data-autoid={`${stablePrefix}--content-block-simple`}
+    customClassName={`${prefix}--content-block-simple ${prefix}--col-lg-8`}>
```

Comment:
Shouldn't use grid in the component.

Reply:
This is contrary to what I've been told by @wonilsuhibm. Are we not dictating what the maximum size of these patterns should be? Otherwise authors can place it in a 10-col if they wanted. The visual designs we use show these patterns laid out in an 8-col grid. Are we missing something?

Reply:
I agree with both perspectives—I'm wondering if we can control how wide or narrow this can be—for example, this pattern should never be in a 4 column layout or 16 column layout. Can we do something like this: we offer two choices: this pattern can be either 8 or 6 cols wide, and let the adopter pick one, instead of letting them specify a random number of columns? And, with this, we do not have an actual need for anything other than 8 columns at the moment—maybe we cross this bridge of multiple options when there is an actual need...? What do you guys think?

Reply:
That type of layout is dictated at the section level. So for instance this will need to sit inside of something like table of contents. That's what will dictate the layout for this component. If we start adding grid elements to this it won't be able to sit where we need it to sit.